### PR TITLE
Upgrade to 2.2.0

### PR DESCRIPTION
**Pull request by JonathonO**
Conflict files:

```
.github/workflows/IntegrationTest.yml
.github/workflows/PerfTest.yml
pom.xml
pom_confluent.xml
src/main/java/com/snowflake/kafka/connector/Utils.java
src/main/java/com/snowflake/kafka/connector/internal/InternalUtils.java
src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionService.java
src/main/java/com/snowflake/kafka/connector/internal/streaming/SchematizationUtils.java
src/main/java/com/snowflake/kafka/connector/records/RecordService.java
```

### CODE REVIEW

**Comment by JonathonO** [comment]
@acristu will have 2.2.0 code changes merged shortly. Can you please review this though and the features we've added?

For reference, here's the conflict file list.

We should have column ordering, debezium type handling and auto schematization changes in some of these classes.

Worth nothing Snowflake introduced special char/reserved keyword handling in this release and a new Utils method for quoting column names. Not sure if it affects your column ordering code?

The `pom*.xml`s also had some version changes (backwards in some cases) and a change in junit dependency. Not sure if that's an issue?

Conflicts:
	.github/workflows/IntegrationTest.yml
	.github/workflows/PerfTest.yml
	pom.xml
	pom_confluent.xml
	src/main/java/com/snowflake/kafka/connector/Utils.java
	src/main/java/com/snowflake/kafka/connector/internal/InternalUtils.java
	src/main/java/com/snowflake/kafka/connector/internal/streaming/SchematizationUtils.java
	src/main/java/com/snowflake/kafka/connector/records/RecordService.java

File: src/main/java/com/snowflake/kafka/connector/records/RecordService.java [hunk 1]
```diff
@@ -413,8 +443,9 @@ public static JsonNode convertToJson(Schema schema, Object logicalValue) {
                 ISO_DATE_TIME_FORMAT.get().format((java.util.Date) value));
           }
           if (schema != null && Time.LOGICAL_NAME.equals(schema.name())) {
-            return JsonNodeFactory.instance.textNode(
-                TIME_FORMAT.get().format((java.util.Date) value));
+            ThreadLocal<SimpleDateFormat> format =
```
Comment (acristu) [inline src/main/java/com/snowflake/kafka/connector/records/RecordService.java:446]: @JonathonO  this looks bad to me ... why instantiate a ThreadLocal as a local variable inside a function ? but I don't think it does any harm, just wasteful ... assume this comes from upstream ...

Comment (JonathonO) [reply src/main/java/com/snowflake/kafka/connector/records/RecordService.java:446]: @acristu yes, this comes from upstream.

Comment (JonathonO) [reply src/main/java/com/snowflake/kafka/connector/records/RecordService.java:446]: Perhaps raise an issue ticket to discuss?

**Comment by JonathonO** [comment]
@acristu any issue with merging this now?

**Pull request merged**